dev/com.ibm.ws.repository.resolver/src/com/ibm/ws/repository/resolver/RepositoryResolutionException.java

```diff
@@ -40,14 +42,18 @@ public class RepositoryResolutionException extends RepositoryException {
     private final Collection<ProductRequirementInformation> missingProductInformation;
     private final Collection<MissingRequirement> allRequirementsResourcesNotFound;
     private final Map<String, Collection<Chain>> featureConflicts;
+    private Set<String> resolvedPlatforms;
+    private Set<String> missingPlatforms;
+    private boolean hasVersionlessIssue;
```

Comment:
`hasVersionlessIssue` doesn't appear to be used anywhere in this class?

Reply:
removed